editor/enable-tracking-prompt/test/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
```

Comment:
I am still having problems getting this to work, specifically with the `it( 'should show and hide a popover when clicking More info' )` test not covered in the above diff.  See https://github.com/WordPress/gutenberg/pull/2278/commits/861218db4398645b338f23c613fa579aa298ed09.